### PR TITLE
Fix incorrect day name and remove duplicated test

### LINKED ISSUE #6014: Wrong day name in dateUtils.test.ts const
Tiniest of nits, but I probably just spent a couple minutes looking at ESLint errors/warnings and noticed that in the tests, if you add 2 days to "Thu Dec 12 2020", you get "Mon Dec 14 2020". The number of days match, but the day name was throwing me. I wasn't sure how we got from Thu=>Mon or if we were somehow calculating business days (but then the day numbers would have been wrong).

Not sure if we want to update that day name to "Sat Dec 12 2020" to avoid potential future confusion. https://www.timeanddate.com/calendar/monthly.html?year=2020&month=12&country=27

https://github.com/mozilla/experimenter/blob/2734dc2cd424497218c1bb25f25de3f5054bfe46/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts#L12

https://github.com/mozilla/experimenter/blob/2734dc2cd424497218c1bb25f25de3f5054bfe46/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts#L35-L39

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1522)

### LINKED ISSUE #6015: Duplicated test in src/lib/dateUtils.test.js
Found via ESLint and local eslint-plugin-jest tinkering:

```
/Volumes/Dev/github/mozilla/experimenter/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts
  41:10  error  Describe block title is used multiple times in the same describe block  jest/no-identical-title
```

https://github.com/mozilla/experimenter/blob/2734dc2cd424497218c1bb25f25de3f5054bfe46/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts#L35-L45

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1523)

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #6014
Fixes #6015

### CODE REVIEW

File: app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts [hunk 1]
```diff
@@ -38,12 +38,6 @@ describe("addDaysToDate", () => {
   });
 });
 
-describe("addDaysToDate", () => {
```
Comment (pdehaan) [inline app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts:41]: This is the same test as L35-39 above. ✂️

File: app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts [hunk 2]
```diff
@@ -38,12 +38,6 @@ describe("addDaysToDate", () => {
...
-  it("should add days to a date and return a date string", () => {
-    expect(addDaysToDate(FAKE_DATE, 2)).toEqual("Mon Dec 14 2020");
```
Comment (pdehaan) [inline app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts:43]: Note that this is the `FAKE_DATE` from in the earlier in the PR diff (although this comment equally applies to L37 above). Note that Dec 12, 2020 was a Saturday, so Saturday + 2 days is Monday. Thursday + 2 days is only a Monday if we don't count weekends. 🤷

**Review by jaredlockhart — APPROVED** [review]
Thnx @pdehaan !

**Comment by jaredlockhart** [comment]
Also commit formatting guidance is [here](https://github.com/mozilla/experimenter/blob/main/contributing.md#git-commit-guidelines) for reference 👍

**Pull request merged**